qubesmanager/qube_manager.py

```diff
@@ -869,6 +940,33 @@ def fill_cache(self):
 
         progress.setValue(row_no)
 
+    def init_template_menu(self):
+        self.template_menu.clear()
+        for vm in self.qubes_app.domains:
+            if vm.klass == 'TemplateVM':
+                action = self.template_menu.addAction(vm.name)
+                action.setData(vm.name)
+                action.triggered.connect(partial(self.change_template, vm.name))
+
+    def _get_default_netvm(self):
+        for vm in self.qubes_app.domains:
+            if vm.klass == 'AppVM':
+                return vm.property_get_default('netvm')
+
+    def init_network_menu(self):
+        default = self._get_default_netvm().name
+        self.network_menu.clear()
+        action = self.network_menu.addAction("None")
+        action.triggered.connect(partial(self.change_network, None))
+        action = self.network_menu.addAction("default ({0})".format(default))
+        action.triggered.connect(partial(self.change_network, 'default'))
```

Comment:
I'd recommend using `qubesadmin.DEFAULT` value instead of string `'default'`. This way you won't confuse it with VM named "default" (if it would be legal), but also you can normally assign it, instead of special handling.

Reply:
I tried using it but I can't show the network name in the confirmation dialog.